pretty/src/pretty/pretty_window.rs

```diff
@@ -0,0 +1,122 @@
+use std::fmt;
+
+use crate::geometry::{Pos, Rect, Region};
+use crate::style::{Shade, Style};
+
+/// A "window" that supports the methods necessary to pretty-print a document.
```

Comment:
/// A "window" that supports the methods necessary to render a set of documents according to a `PaneNotation`[link].

Reply:
Leaving out the reference to PaneNotation because they don't exist yet in this PR.